Misc/NEWS.d/next/Core and Builtins/2018-09-13-11-14-38.bpo-34663.BtJz-5.rst

```diff
@@ -0,0 +1,2 @@
+Added support for `usevfork` parameters of `posix_spawn`, which allows to
```

Comment:
use_vfork with underscore, no?

`posix_spawn` can be written :func:`os.posix_spawn` to get a link to the function in the generated doc.